README.md

```diff
@@ -1267,90 +1267,132 @@ The `job-id-regex` should contain one capture group while matching against the w
 
 Allows to execute jobs using HTCondor which is a specialized workload management system for compute-intensive jobs created by the Center for High Throughput Computing in the Department of Computer Sciences at the University of Wisconsin-Madison (UW-Madison).
 
-This backend creates six files in the `<call_dir>` (see previous section):
+The backend is specified via the actor factory `ConfigBackendLifecycleActorFactory`:
 
-* `script` - A shell script of the job to be run.  This contains the user's command from the `command` section of the WDL code.
-* `stdout` - The standard output of the process
-* `stderr` - The standard error of the process
-* `submitfile` - A submit file that HtCondor understands in order to submit a job
-* `submitfile.stdout` - The standard output of the submit file
-* `submitfile.stderr` - The standard error of the submit file
+```
+backend {
+  providers {
+    HtCondor {
+      config {
+        actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+        # ... other configuration
+      }
+    }
+  }
+}
+```
 
-The `script` file contains:
+This backend makes the same assumption about the filesystem that the local backend does: the Cromwell process and the jobs both have read/write access to the CWD of the job.
```

Comment:
I'd call this superfluous but I'll fall back to the "we'll clean up the docs later" thing. There's now a ticket in the dsde-docs repo to fix up all the config backend docs.